app/globals.css

```
@@ -61,6 +61,7 @@
   --font-sans: var(--font-geist-sans), ui-sans-serif, system-ui, sans-serif;
   --font-mono: var(--font-geist-mono), ui-monospace, SFMono-Regular, Menlo, monospace;
   --font-serif: var(--font-fraunces), ui-serif, Georgia, "Times New Roman", serif;
+  --font-hanken: "Avenir Next", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Avenir" to be "avenir" (value-keyword-case)

(value-keyword-case)